packages/api/src/domain/medical/patient.ts

```diff
@@ -70,6 +70,7 @@ export type PatientData = {
   externalData?: PatientExternalData;
   cxDocumentRequestMetadata?: unknown;
   cxConsolidatedRequestMetadata?: unknown;
+  whOverride?: boolean;
```

Comment:
I think we should store this on each one of the props for docQuery and consolidated, so triggering a doc query disabling WH doesn't mess w/ the cx's consolidated one.

Could just store inside the unknown property?